notebooks/benchmarking.md

```diff
@@ -0,0 +1,404 @@
+# Benchmarking the Invisible Maps Algorithm
+
+We've implemented a few datasets to evaluate the effectiveness of the g2o algorithm behind invisible maps' optimization. 
+
+## Definitions:
+
+"Ground truth metric" is computed as follows:
+
+Calculates the transforms from the anchor tag to each other tag for the optimized and the ground truth tags, then compares the transforms and finds the difference in the translation components.
+
+## Workflow
+There are a number of different "weights" that can be used that value certain elements of the map more highly when optimizing the map. There's additionally an option to parameter sweep, which (to us) is a bit of a black box that we're just assuming sweeps every relevant parameter(?). What we're looking at here is the ground truth metric at pre-optimization and comparing it to the ground truth metric the parameter sweep finds as optimal.
```

Comment:
Can we say anything more definitive at this point (now that you've gone into the code in more detail)?  It would be nice to not have to say things like "we think" or "it's a black box" when it's code that our group wrote.

Reply:
Deleting this file! I think it's a relic since most of the files that we ended up testing on were not the ones specified in this document